assets/css/style.css

```diff
@@ -776,10 +783,22 @@ ul.share-buttons .sr-only{
 
 .people{
 	display: grid;
-	grid-template-columns: 24% 24% 24% 24%;
+	grid-template-columns: repeat(2, 1fr);
 	grid-gap: 20px;
 }
 
+@media (min-width: 768px) {
```

Comment:
These are fairly classic tablet and desktop breakpoints, but happy to use a different set if folks prefer. I didn't think we needed to introduce quite as many as something like [Tailwind](https://tailwindcss.com/docs/responsive-design#overview), so I opted to use the two most common ✅